script.js

```diff
@@ -0,0 +1,12 @@
+function sum() {
+    let a = document.getElementById("firstNumber");
+    let b = document.getElementById("secondNumber");
+    let aNum = parseInt(a.value);
+    let bNum = parseInt(b.value);
+
+    let sum = aNum + bNum;
```

Comment:
We'll discuss the use of `let` in a future class, it works here but when we do you'll see why you wouldn't use it.